src/packet_analysis/protocol/ip/IP.cc

```diff
@@ -11,6 +11,10 @@
 #include "zeek/Event.h"
 #include "zeek/TunnelEncapsulation.h"
 
+// XXX: This is only for zeek::analyzer::Analyzer::GetIgnoreChecksumsNets()
```

Comment:
XXX?

Reply:
Only responding here, happy to address the other comments depending on thoughts on the following.

The reason for that XXX was that it felt somewhat wrong to have the dependency from `src/packet_analysis` into the `src/anayzer` (`zeek::packet_analysis` to `zeek::analzyer`).

One approach to decouple could be to introduce a separate class (ConfigManager ?) that gets the helpers and can be used from both `zeek::packet_analysis` and `zeek::analyzer`. But not sure that's overdoing it if there's never another `option` that is accessed so often that it makes sense to "cache" it. 

```
static void SetIgnoreChecksumsNets(TableValPtr t);
static TableValPtr GetIgnoreChecksumsNets(void);
```